<!-- %Tag(FULL)%-->
<launch>


 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="xbox_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.1" />
    <param name="autorepeat_rate" value="20" />
    <remap from="/joy" to="/arm_joy" />
  </node>

 <!-- Axes -->
  <param name="SlewAxis" value="0" type="int"/>
  <param name="SlewDir" value="1" type="int"/>
  <param name="ShoulderAxis" value="1" type="int"/>
  <param name="ShoulderDir" value="1" type="int"/>
  <param name="ElbowAxis" value="4" type="int"/>
  <param name="ElbowDir" value="-1" type="int"/>
  <param name="JawRotateAxis" value="3" type="int"/>
  <param name="JawCloseAxis" value="5" type="int"/>
  <param name="WristDir" value="-1" type="int"/>
  <param name="JawOpenAxis" value="2" type="int"/>
  <param name="JawDir" value="1" type="int"/>
  <param name="scale" value="2500" type="double"/>
  
  <param name="LockSlewButton" value="0" type="int"/>  
  <param name="LockJawRotateButton" value="1" type="int"/>  
  <param name="LockJawOpenButton" value="2" type="int"/>
  


  <node pkg="arm5_joy_control" type="arm5_joy_control" name="arm5_joy_control" respawn="false" output="screen"/>

</launch>
<!-- %EndTag(FULL)%-->
